ding/policy/sac.py

```diff
@@ -162,6 +162,7 @@ class SACPolicy(Policy):
             # You can use either "n_sample" or "n_episode" in actor.collect.
             # Get "n_sample" samples per collect.
             # Default n_sample to 1.
+            use_logit=False,
```

Comment:
add comment for this argument，and our variable name should be like `use_xxx`, `is_xxx`，`collector_logit` can be better